NitroxServer/Communication/Packets/Processors/EntitySpawnedByClientProcessor.cs

```diff
@@ -35,14 +35,14 @@ public override void Process(EntitySpawnedByClient packet, Player playerWhoSpawn
             if (entity is WorldEntity worldEntity)
```

Comment:
With the recent changes in entities system, we might need to double check there's not others specific check like that

Reply:
Couldn't find similar code at the moment, if there are any similar issues it'll appear sooner or later with 1.8 reports though.